src/components/20-molecules/dropdown/ui.test.js

```diff
@@ -6,48 +6,41 @@ fixture('Dropdown').page(`${host}/iframe.html?id=molecules-dropdown--dropdown`);
 
 test('should render correctly', async t => {
   const dropdown = await Selector(() =>
-    document.querySelector(`axa-dropdown[data-test-id="dropdown"]`)
+    document.querySelector(`axa-dropdown`)
   );
   await t.expect(dropdown.exists).ok();
 });
 
-fixture('Dropdown valid').page(
-  `${host}/iframe.html?id=molecules-dropdown--dropdown-w-valid-checkmark`
+fixture('Dropdown check mark').page(
+  `${host}/iframe.html?id=molecules-dropdown--dropdown&knob-checkmark=true`
 );
 test('should show checkmark', async t => {
   const dropdownCheckmark = await Selector(() =>
     document.querySelector(
-      `axa-dropdown[data-test-id="dropdown-valid"] .m-dropdown__valid-icon .m-dropdown__valid-icon-inner-active`
+      `axa-dropdown .m-dropdown__check`
```

Comment:
I would recommend using `js-dropdown__check` instead of `m-dropdown__check`.

Reply:
good catch! Done.